dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionMetadata.java

```diff
@@ -112,6 +112,9 @@
     @Schema(type = "integer", format = "int64")
     private Timestamp dbUpdateDate;
 
+    @Column()
```

Comment:
Based on the criteria discussions we've had, I'd say a boolean is sufficient to provide metrics to endusers.

More information might be useful though, such as: 
1. Breakdown by protocol (as Denis suggested)
2. Which URLs weren't accessible. This would be a QoL improvement for debugging and for endusers switching out 'bad' URLs.

But, those may not be worthwhile at this point... not sure.